pkg/test/config/toolchainconfig.go

```diff
@@ -85,17 +85,20 @@ func (o AutomaticApprovalOption) Enabled(value bool) AutomaticApprovalOption {
 	return o
 }
 
+// Deprecated: This is no longer used for anything.
```

Comment:
nice - thanks for adding those comments. I guess you'll delete those in follow up PRs, once we switch to the new ones ?

Reply:
Yes, that's the intention... There's already an issue for that: https://issues.redhat.com/browse/KUBESAW-36